src/features/transfer-ownership/TransferOwnership.module.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../../node_modules/@zero-tech/zui/styles/_colors.scss';
+@import '../../../node_modules/@zero-tech/zui/styles/_typography.scss';
+
+.Container {
+	width: 36.25rem;
+	padding: 3rem;
+
+	.ErrorMessage {
+		display: block;
+		color: $helitrope;
+		font-size: $size-small;
+		font-weight: $font-weight-bold;
+		margin-top: 0.75rem;
+		text-align: center;
+	}
```

Comment:
Hmm perhaps we need to build a `Wizard.Error` component. This is all good for now though.

Reply:
oo yeah that would be v.handy! Also ```Wizard.Details``` would be great, I'll need that a few times for zApp-NFTS